code/game/machinery/computer/operating_computer.dm

```diff
@@ -3,12 +3,13 @@
 
 /obj/machinery/computer/operating
 	name = "operating computer"
-	desc = "Monitors patient vitals and displays surgery steps. Can be loaded with surgery disks to perform experimental procedures. Automatically syncs to operating tables within its line of sight for surgical tech advancement."
```

Comment:
i'd change this to tables or stasis beds

Reply:
fixed